Assignment6/app.py

```
@@ -0,0 +1,207 @@
+            'stock': cached_data['stock'],
+            'from_cache': True
+        }
+    
+    # If not in cache, fetch from API
+    company_data = None
+    stock_data = None
+    
+    try:
+        # Fetch company outlook data
+        company_url = f"https://api.tiingo.com/tiingo/daily/{ticker}?token={TIINGO_API_KEY}"
+        company_response = requests.get(company_url)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)